app/src/main/java/com/android/halanvendordiscovery/domain/vendors/repo/VendorRepository.kt

```diff
@@ -0,0 +1,17 @@
+package com.android.halanvendordiscovery.domain.vendors.repo
+
+import com.android.halanvendordiscovery.domain.vendors.model.Category
+import com.android.halanvendordiscovery.domain.vendors.model.Page
+import com.android.halanvendordiscovery.domain.vendors.model.PagedList
+import com.android.halanvendordiscovery.domain.vendors.model.Vendor
+
+interface VendorRepository {
+
+    suspend fun getVendors(
+        pageConfig: Page,
+        categoryId: String? = null,
```

Comment:
Why categoryId can be nullable?

Reply:
because you need to show a list of all available vendors for the user before selecting any category